groceries/schema.sql

```diff
@@ -3,6 +3,8 @@ DROP TABLE IF EXISTS items;
 CREATE TABLE items (
   id INTEGER PRIMARY KEY AUTOINCREMENT,
   name TEXT UNIQUE NOT NULL
+--   needed BOOLEAN,
+--   category TEXT UNIQUE NOT NULL
```

Comment:
When we implement this, I believe it will be a `category_id`, that points to the ID field on the categories table.

Regardless, I don't think we should commit these comments, since the work is already captured in the card.